src/main/views/macro/table.njk

```diff
@@ -20,7 +20,12 @@
     </div>
     {% if changeLink %}
       <div class="column-one-quarter change-link-column">
-        <a href="{{ changeLink | safe }}" class="bold">{{ t('Change') }} <span class="visuallyhidden">{{ label | lower }}</span></a>
+        {% if (label === 'Full name') %}
+          <a href="{{ changeLink | safe }}" class="bold">{{ t('Change') }} <span class="visuallyhidden">{{ 'your full name, address and correspondence address' | lower }}</span></a>
```

Comment:
This is not just one place we need to update. Noticed there are other sections that need similar fix on the same page. Below are few details where I noticed the issue.

- Claimant > check and submit > Their Details (this also has similar bug)
-  Claimant > check and submit > Claimant Amount
- Defendant > journey > YET TO BE LOOKED INTO
- For Defendant we need to check for 'Individual', 'Company', .... etc as well

Reply:
It's not a good idea to refactor this in DAC ticket, we can have a bug raised and take this one. I will create a bug for this one as this is an existing code.

Reply:
Created the bug ROC-8242 for this issue.

JIRA URL : https://tools.hmcts.net/jira/browse/ROC-8242